web/packages/teleport/src/config.ts

```diff
@@ -379,6 +383,19 @@ const cfg = {
 
   playable_db_protocols: [],
 
+  getNonExactRoutes() {
+    // These routes will not be exact matched when deciding if it is a valid route
+    // to redirect to when a user is unauthenticated.
+    // This is useful for routes that can be infinitely nested, e.g. `
+    // /web/accessgraph` and `/web/accessgraph/integrations/new`
+    // (`/web/accessgraph/*` wouldn't work as it doesn't match `/web/accessgraph`)
+
+    return [
+      // at the moment, only `e` has an exempt route, which is merged in on `cfg.init()`
+      ...this.nonExactRoutes,
```

Comment:
why `this` instead of `cfg` like the rest of the methods? also, why the spread?

Reply:
No idea, I'll change

The spread was so future diffs are just adding a new line to this with the OSS routes

Reply:
Oh ok that makes sense (and the comment explains it too idk why I missed it)